tree/ntuple/v7/src/RNTupleMerger.cxx

```diff
@@ -558,13 +558,33 @@ static void ExtendDestinationModel(std::span<const RFieldDescriptor *> newFields
         (newFields.size() > 1 ? "them" : "it"), mergeData.fNumDstEntries);
 
    changeset.fAddedFields.reserve(newFields.size());
+   // First add all non-projected fields...
    for (const auto *fieldDesc : newFields) {
+      if (!fieldDesc->IsProjectedField()) {
+         auto field = fieldDesc->CreateField(*mergeData.fSrcDescriptor);
+         changeset.AddField(std::move(field));
+      }
+   }
+   // ...then add all projected fields.
+   Internal::RProjectedFields::FieldMap_t fieldMap;
```

Comment:
This does not look right, we need to have one `fieldMap` per projected field! I guess the variable should move into the loop, you probably need a second source with two projections as a test.